model_downloader/list_topologies.yml

```diff
@@ -1839,7 +2120,10 @@ topologies:
     license: https://raw.githubusercontent.com/opencv/open_model_zoo/master/LICENSE
 
   - name: "text-detection-0002-fp16"
-    description: "Detects oriented text."
+    description: >-
+      Text detector based on [PixelLink](https://arxiv.org/pdf/1801.01315.pdf) architecture
```

Comment:
IMO, the Markdown link syntax doesn't look natural as plain text. I suggest you transform it into something more plain, e.g. "PixelLink \<https\://arxiv.org/pdf/1801.01315.pdf\>".

Reply:
@IRDonch 
resolved